src/UpdatableTimer/UpdatableTimer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+namespace TemporalioSamples.UpdatableTimer;
+
+public class UpdatableTimer(DateTimeOffset wakeUpTime)
```

Comment:
Optional neat improvement, support providing a summary for the timer that appears in the UI (there is a `WaitConditionAsync` overload that accepts options now which includes a summary)